### PR TITLE
Morse Code Decoder

### PR DESCRIPTION
**Pull request by tomasesquivelgc**
# In this branch:
 - We create our first Ruby project complete with linters.
 - We create a Morse Code Decoder using said language.
 - 3 separate methods (as per assignment) each corresponding to letter, word, and sentence decoding.

### CODE REVIEW

**Review by Lucas-Masaba — CHANGES_REQUESTED** [review]
# Hello @tomasesquivelgc && @waelelsafty07  👋 , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request ⚠️ 
- [ ] @waelelsafty07 only contributed to the README and not the actual codebase in this milestone. You should all equally contribute to the actual codebase.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏 💻 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @Lucas-Masaba in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: README.md [hunk 1]
```diff
@@ -1,2 +1,171 @@
...
+
+```sh
+  cd my-project
+  npm install
+```
+
+### Usage
+
+To run the project, execute the following command:
+
```
Comment (LionRouge1) [inline README.md]: - [x] Nice work filling the get started section :clap: However the information under it doesn't correspond to this project. Please update your README file with the appropriate command to run this project

File: morse_decoder.rb [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    '--' => 'M', '-.' => 'N', '---' => 'O',
+    '.--.' => 'P', '--.-' => 'Q', '.-.' => 'R',
+    '...' => 'S', '-' => 'T', '..-' => 'U',
+    '...-' => 'V', '.--' => 'W', '-..-' => 'X',
+    '-.--' => 'Y', '--..' => 'Z'
+  }.freeze
+
+  def self.decode_char(morse_char)
+    MORSE_CODE_DICT.key(morse_char)
+  end
```
Comment (LionRouge1) [inline morse_decoder.rb]: - [x] Nice work saving the characters in a constant variable :+1: However the  method `decode_char` for decoding a single character is returning `nil`. Kindly make sure the method return the value of key entered. 
**HIT:** `MORSE_CODE_DICT[morse_char]`

**Review by LionRouge1 — CHANGES_REQUESTED** [review]
Hi **Team**, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost

## Highlights
- Create a method named `decode_char` :heavy_check_mark: 
- Create a method named `decode_word` :heavy_check_mark: 
- All linters passed :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by KanzaTahreem — APPROVED** [review]
# Approved 🙌

Hi @tomasesquivelgc & @waelelsafty07 

Good job implementing the suggested changes 💪 **_Your project is COMPLETE! There is nothing else to say other than... it's time to merge it! 🚀 Congratulations!_**



## Highlights 🎯
There are some excellent points that you have made perfectly. Keep up the great work! 👏

-  ✅ Message is decoded well
-  ✅ Good use of DRY
-  ✅ Your README looks good
-  ✅ Linters are working properly

## Optional suggestions ❕
N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**